languages/csharp/exercises/concept/numbers/.docs/introduction.md

```diff
@@ -14,4 +14,19 @@ C# has two types of numeric conversions:
 
 As an `int` has less precision than a `double`, converting from an `int` to a `double` is safe and is thus an implicit conversion. However, converting from a `double` to an `int` could mean losing data, so that requires an explicit conversion.
 
-In this exercise you must conditionally execute logic. The most common way to do this in C# is by using an `if/else` statement.
+In this exercise you must conditionally execute logic. The most common way to do this in C# is by using an `if/else` statement:
+
+```csharp
```

Comment:
Iff  samples should be minimally viable, and for clarity on the types used here, perhaps add a first line of `int x = 6`. This particular value of `x` may also draw students to more fully understand the syntax's use.

Reply:
I like the suggestion! Added it.